ts/input/tex/color/ColorMethods.ts

```diff
@@ -121,14 +121,13 @@ ColorMethods.DefineColor = function (parser: TexParser, name: string) {
  * @param {string} name The name of the control sequence.
  */
 ColorMethods.ColorBox = function (parser: TexParser, name: string) {
-  const cname = parser.GetArgument(name);
+  const model = parser.GetBrackets(name, "");
```

Comment:
Note that our coding style uses single quotation marks, not double, for string constants (tslint should flag this).  This occurs several times below as well.